File: Python/Functions/palindrome.py
value=input("Enter a word ")

def palindrome(x):
    left = 0
    right= len(x)-1

    while right>=left:
        if x[left]==x[right]:
            left=left+1
            right=right-1
        else:
            return False
    return True
print("Check for palindrome: ")
print(palindrome(value))
    
    

File: JavaScript/palindrome.js
var string = prompt("Please enter any word or number.");

function palindrome(MyString) {
    // Remove non-alphanumeric characters and convert to lowercase
    var cleanedString = MyString.replace(/[^A-Z0-9]/ig, "").toLowerCase();
    
    // Reverse the cleaned string
    var reversedString = cleanedString.split("").reverse().join("");
    
    // Check if the reversed string is the same as the original cleaned string
    if (reversedString === cleanedString) {
        document.write(MyString + " is a palindrome.");
    } else {
        document.write(MyString + " is not a palindrome.");
    }
}

palindrome(string);
